src/Dhall/Import.hs

```diff
@@ -540,13 +540,14 @@ getCacheFile hash = do
                     liftIO (Directory.setPermissions directory private)
 
     let tryXdgCacheHome = do
-            Just cacheDirectory <- liftIO (System.Environment.lookupEnv "XDG_CACHE_HOME")
+            cacheDirectory <- liftIO $ Directory.getXdgDirectory Directory.XdgCache ""
+            
             assertDirectory cacheDirectory
 
             return cacheDirectory
 
     let tryHome = do
```

Comment:
@jneira: You can delete the `tryHome` alternative since `Directory.getXdgDirectory` already falls back to `~/.cache` if the `XDG_CACHE_HOME` environment variable is unset